website/docs/r/function_trigger.html.markdown

```diff
@@ -94,6 +97,7 @@ The following arguments are supported:
 The following attributes are exported:
 
 * `id` - The ID of the new trigger.
```

Comment:
Change all resource docs to say the id is combination of namespace/respectiveresourceID
Add respective new attribute like "action_id"..so on 

Add a import section block in resources blocks which describes how you do import ...Refer some old docs like is_lb_pool docs for more etails